test/solveEquation/solveEquation.test.js

```diff
@@ -58,6 +58,12 @@ describe('solveEquation for =', function () {
     ['2(x+3)/3 = 2', 'x = 0'],
     ['( u )/( 0.3) = 4u + 6.28', 'u = -9.42'],
     ['- q - 4.36= ( 2.2q )/( 1.8)', 'q = -1.962'],
+    //['2/x = 1', 'x = 2'], TODO: fixme
+    //['2/(4x) = 1', 'x = 1/2'], TODO: bombs out
+    //['2/(8 - 4x) = 1/2', 'x = 1'], TODO: fixme
+    //['2/(1 + 1 + 4x) = 1/3', 'x = 1'], TODO: fixme
+    //['(3 + x) / (x^2 + 3) = 1', '-x^2 + x = 0'], TODO: bombs out
+    //['6/x + 8/2x = 10', '(-10x) + 4x^2 = -6'], TODO: fixme
```

Comment:
this is an interesting case where    `x * 6/x` isn't multiplied to get `(x * 6) / x`, which would cancel out

one solution is just to have better cancelling code that doesn't need it to be a fraction.

another (easier for now) solution would be to add a case for a symbol times a symbol in the denominator (i.e. we don't want `9/2 * x` to multiply to `9x / 2`, but we would want `9/x * x` to multiply to `9`)

... anyways I'll open an issue  #139 for this too

(btw `8/2x` is parsed as `8/2 x` ie `4x`, not as `8 / 2x`)